addons/project/models/project_collaborator.py

```diff
@@ -30,10 +30,9 @@ def create(self, vals_list):
         return project_collaborators
 
     def unlink(self):
+        self.project_id.task_ids.message_unsubscribe(self.partner_id.ids)
         res = super().unlink()
-        # Check if it remains at least a collaborator in all shared projects.
-        collaborator = self.env['project.collaborator'].search([], limit=1)
-        if not collaborator:  # then disable the project sharing feature
+        if res:
```

Comment:
Isn't `unlink` supposed to always return True? I don't get this condition.

Reply:
I agree with Audric. 🤔 

Also, I did not understand that change, it does not seem we ask to do that in the task.

May I miss something?

Reply:
You are correct; `unlink` always returns True. This is a very old pull request, and it seems these changes were not relevant. I have reverted these changes as well.